cultivation/scripts/running/run_performance_analysis.py

```
@@ -60,45 +153,78 @@
     args = parser.parse_args()
 
+    # --- Load wellness context if available ---
+    wellness_context = None
+    VERBOSE = False
+    try:
+        wellness_path = os.path.join(os.path.dirname(__file__), '../../data/daily_wellness.parquet')
+        wellness_path = os.path.abspath(wellness_path)
+        if os.path.exists(wellness_path):
+            wellness_df = pd.read_parquet(wellness_path)
+            # Use today's date or closest available
+            today = df.index[0].date() if hasattr(df.index[0], 'date') else pd.to_datetime(df.index[0]).date()
```

Ruff (0.8.2)

Undefined name `df`

(F821)